conbench/db.py

```diff
@@ -38,7 +38,11 @@ def configure_engine(url):
         # tickets and discussions:
         # https://github.com/conbench/conbench/issues/599
         # https://github.com/conbench/conbench/pull/690
-        connect_args={"options": "-c timezone=utc -c statement_timeout=60s"},
+        # https://docs.sqlalchemy.org/en/20/core/engines.html#use-the-connect-args-dictionary-parameter
+        connect_args={
+            "options": "-c timezone=utc -c statement_timeout=60s",
+            "connect_timeout": 3,
```

Comment:
Is this 3 seconds? Or some other unit? Not a big deal, just curious (I did look in the URL up above, but couldn't find what I was looking for either with a quick scan)

Reply:
Would you mind adding a small comment? Agreed with Jon that it's hard to find.

```suggestion
            "connect_timeout": 3,  # seconds
```

Reply:
Yes! Adding a commit.